cli/testdata/server-config.yaml.golden

```diff
@@ -232,6 +232,18 @@ introspection:
     # Allow administrators to enable Terraform debug output.
     # (default: false, type: bool)
     enableTerraformDebugMode: false
+  healthcheck:
+    # Overall timeout for health checks.
+    # (default: 30s, type: duration)
+    timeout: 30s
```

Comment:
Do we need to expose this configuration option? Wouldn't it be enough to keep it hardcoded for now?

Reply:
https://github.com/coder/coder/issues/8969 exists to expose recheck interval and I figured that the timeout would also be relevant. I don't have any strong opinions either way, so if you'd prefer it not exposed we can keep it hard-coded at 30s.

Reply:
Is there any risk of misconfiguring it? Let's say an operator will set it to 5 mins.

Reply:
OK, keeping timeout hard-coded for now. We can expose it if someone asks.